src/Agent.Service/Windows/Program.cs

```diff
@@ -12,33 +14,37 @@ static class Program
         /// </summary>
         static int Main(String[] args)
         {
-            if (args != null && args.Length == 1 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
+            if (args != null && args.Length >= 1)
             {
-                // TODO: LOC all strings.
-                if (!EventLog.Exists("Application"))
-                {
-                    Console.WriteLine("[ERROR] Application event log doesn't exist on current machine.");
-                    return 1;
-                }
+                EventLogger.WriteInfo(String.Format("Received Command - {0}", args[0]));
 
-                EventLog applicationLog = new EventLog("Application");
-                if (applicationLog.OverflowAction == OverflowAction.DoNotOverwrite)
+                if(args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
                 {
-                    Console.WriteLine("[WARNING] The retention policy for Application event log is set to \"Do not overwrite events\".");
-                    Console.WriteLine("[WARNING] Make sure manually clear logs as needed, otherwise AgentService will stop writing output to event log.");
+                    return SetupEventSource();
                 }
 
-                try
+                if(args[0].Equals("runAsProcess", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Actually I'm thinking that "AutoLogon" may be better than "InteractiveService". Service means something very specific (e.g. Windows Service) and I worry it will lead to confusion to start reusing the word "Service".